### PR TITLE
revert spring upgrade and suppress some CVE warnings

### PR DESCRIPTION
**Pull request by haileyajohnson**

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -247,10 +247,10 @@ libraries["jaxb-api"] = "javax.xml.bind:jaxb-api:2.3.1"
 
 libraries["jaxen"] = "jaxen:jaxen:1.1.4"
 
-libraries["threddsIso"] = "EDS:threddsIso:2.2.20-SNAPSHOT"
+libraries["threddsIso"] = "EDS:threddsIso:2.2.18"
```
Comment (tdrwenski) [inline gradle/dependencies.gradle]: why downgrade `threddsIso`?

**Review by tdrwenski — COMMENTED** [review]
I think you may want to revert my upgrade to hibernate and the validation api as well (commit 4e559177d9)

**Review by tdrwenski — APPROVED** [review]

**Comment by srstsavage** [comment]
@haileyajohnson Is there a timeline on when these fixes will be released, I assume as 4.6.22? I went to create a 4.6.21 Docker image and spent some time down the rabbit hole trying to figure out the dependency errors...

**Comment by haileyajohnson** [comment]
You can get a working 4.6.22-SNAPSHOT currently from our [downloads page](https://downloads.unidata.ucar.edu/tds/).
To be honest, I don't expect to ever have a full 4.6.22 release or an upgrade to Spring 5, as TDS 4.6.x will be fully deprecated next month.

**Comment by srstsavage** [comment]
Ok thanks, good to know!

**Pull request merged**